### ISSUE TITLE
Errors for invalid formatting use weird characters for spacing

### ISSUE THREAD

**Issue by Tyrrrz** [belav/csharpier#947]
CSharpier appears to use `┬╖┬╖┬╖┬╖┬╖┬╖┬╖┬╖` characters to space out the code snippets in the terminal:

![image](https://github.com/belav/csharpier/assets/1935960/e18b12ad-f6c6-496e-817b-9b74e2e4584d)

It makes the error quite hard to parse visually, and I don't think this is the intended behavior. Any idea why this may be happening?

**Comment by belav** [comment]
CSharpier uses the following so that you can visually see the whitespace when code isn't formatted
```c#
    private static string? MakeWhiteSpaceVisible(string? value)
    {
        return value?.Replace(' ', '·').Replace('\t', '→');
    }
```

The goal is to make it obvious if the reason it failed the check is due to whitespace differences. A failure due to extra whitespace like below isn't obvious if you can't see it.
```c#
public class ClassName
{
····private string field1;
····
····private string field2;
}
```

Where are you seeing this output? CSharpier is writing on stdout using UTF8 encoding.

**Comment by Tyrrrz** [comment]
I'm seeing this both in GH Actions, as well as locally in Rider's integrated terminal.

Here's the link to the actions job where you can see it: https://github.com/Tyrrrz/YoutubeExplode/actions/runs/6026195051/job/16349413554

**Comment by belav** [comment]
Took me a bit - I figured out this is an issue with CSharpier.Msbuild but I believe only on windows.

![image](https://github.com/belav/csharpier/assets/3606529/7dac1322-fa19-4aa6-9195-72d77fa0f134)

**Comment by Tyrrrz** [comment]
A bit related but slightly off-topic, may I suggest displaying the space indicator characters only for indentation (i.e. before the first non-whitespace character)? It gets a bit noisy when it's in the middle of the line (e.g. `return·value?.Replace(...` in your example).

**Comment by belav** [comment]
@Tyrrrz I can play around with limiting where whitespace is visible. Maybe something like just making the first couple of spaces/tabs that are different will work better and not be so noisy. I wrote up #953 and have a couple of ideas to try out for improving it. Thanks for the suggestion!
